### PR TITLE
Fixing linters error

### PR DESCRIPTION
**Pull request by hush-hush**
### What does this PR do?

Fixing linters error

### CODE REVIEW

**Review by ogaca-dd — APPROVED** [review]
👍

**Comment by ogaca-dd** [comment]
Do you know why these errors are not caught by our CI?

File: pkg/trace/obfuscate/cache.go [hunk 1]
```diff
@@ -56,14 +56,14 @@ func newMeasuredCache() *measuredCache {
 	cfg := &ristretto.Config{
 		// We know that the maximum allowed resource length is 5K. This means that
 		// in 5MB we can store a minimum of 1000 queries.
-		MaxCost: 5_000_000,
```
Comment (gbbr) [inline pkg/trace/obfuscate/cache.go:59]: ☹️ Why? This is much nicer and more readable.

**Review by ahmed-mez — APPROVED** [review]

File: pkg/trace/obfuscate/cache.go (hunk 1 above)
Comment (hush-hush) [reply pkg/trace/obfuscate/cache.go]: You were right, this is an issue with my gofmt not being up to date with go 1.13. removing

**Review by gbbr — APPROVED** [review]

**Pull request merged**